model/cluster_request.go

```diff
@@ -31,6 +31,36 @@ type CreateClusterRequest struct {
 	VPC                    string                         `json:"vpc,omitempty"`
 }
 
+func (request *CreateClusterRequest) setUtilityDefaults(utilityName string) {
+	reqDesiredUtilityVersion, ok := request.DesiredUtilityVersions[utilityName]
+	if !ok {
+		request.DesiredUtilityVersions[utilityName] = DefaultUtilityVersions[utilityName]
+		return
+	}
+	if reqDesiredUtilityVersion.Chart == "" {
+		reqDesiredUtilityVersion.Chart = DefaultUtilityVersions[utilityName].Chart
+	}
+	if reqDesiredUtilityVersion.ValuesPath == "" {
+		reqDesiredUtilityVersion.ValuesPath = DefaultUtilityVersions[utilityName].ValuesPath
+	}
+}
+
+func (request *CreateClusterRequest) setUtilitiesDefaults() {
+	for _, utilityName := range []string{
+		PrometheusOperatorCanonicalName,
+		ThanosCanonicalName,
+		NginxCanonicalName,
+		NginxInternalCanonicalName,
+		FluentbitCanonicalName,
+		TeleportCanonicalName,
+		PgbouncerCanonicalName,
+		StackroxCanonicalName,
+		KubecostCanonicalName,
+	} {
+		request.setUtilityDefaults(utilityName)
+	}
```

Comment:
Could we just iterate through the `DefaultUtilityVersions` map keys?
```suggestion
	for utilityName, _ := range DefaultUtilityVersions {
		request.setUtilityDefaults(utilityName)
	}
```

Reply:
why yes, that does make more sense :facepalm: